core/domain/customization_args_util.py

```diff
@@ -118,8 +120,9 @@ def validate_customization_args_and_values(
                 schema_utils.normalize_against_schema(
                     customization_args[ca_spec.name]['value'],
                     ca_spec.schema))
-        except Exception:
+        except Exception as e:
             # TODO(sll): Raise an actual exception here if parameters are
```

Comment:
Do we still need this TODO?

Reply:
Yes, since we will need to remove the flag once audit + migration is complete and then raise exception by checking the params.